src/createServer.js

```diff
@@ -1,3 +1,40 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { validateUrl } = require('./validateUrl');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+    const { pathname, searchParams } = normalizedUrl;
+
+    const text = pathname.slice(1);
+    const toCase = searchParams.get('toCase');
+
+    const sendResponse = (code, message, data) => {
+      res.statusCode = code;
+      res.statusMessage = message;
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify(data));
+    };
```

Comment:
Good idea, I think if you'll create a factory of such helpers it will be more useful on big projects